packages/devtools_app/lib/src/inspector/layout_explorer/ui/widgets_theme.dart

```diff
@@ -0,0 +1,238 @@
+import 'package:flutter/material.dart';
+
+class WidgetTheme {
+  const WidgetTheme({
+    this.iconAsset,
+    this.color = otherWidgetColor,
+  });
+
+  final String iconAsset;
+  final Color color;
+
+  static WidgetTheme fromName(String widgetType) {
+    if (widgetType == null) {
+      return const WidgetTheme();
+    }
+
+    return themeMap[_stripBrackets(widgetType)] ?? const WidgetTheme();
+  }
+
+  /// Strips the brackets of the widget
```

Comment:
dartdoc nit. r/of/off and add new line after first sentence.

```
/// Strips the brackets off of the widget name.
///
/// For example: `AnimatedBuilder<String>` -> `AnimatedBuilder`.
```

Reply:
Done